lib/engines/content_object_store/app/models/content_object_store/content_block_edition.rb

```diff
@@ -0,0 +1,2 @@
+class ContentObjectStore::ContentBlockEdition < ApplicationRecord
```

Comment:
Are we going to add the relations in the next PR?

Reply:
For the Edition I've added the `references` in the `content_block_edition` migration, and was thinking that a future PR could handle any logic around validating the presence of the Document (like the Identifiable concern does in the existing code).
In this PR should we add the relation to the Document class, something like this?
```
  has_many :content_block_editions,
           -> { order(created_at: :asc, id: :asc) },
           inverse_of: :content_block_document
```

Reply:
Yeah, that should do the trick 👍